### ISSUE TITLE
Local version of GZ can't find DECaLS DR2 id

### ISSUE THREAD

**Issue by willettk** [zooniverse/Galaxy-Zoo#256]
I'm trying to get a local version of Galaxy Zoo on my machine so I can try to help with #255. Installation goes fine once the proper version of node is installed, but it throws the following error when trying to load the page in the browser: 

`application.js:22456 Uncaught TypeError: Cannot read property 'id' of undefined`

Relevant bit of the js: 

```
      },
      decals_dr2: {
        id: Config.surveys.decals_dr2.id,
        workflowId: Config.surveys.decals_dr2.workflowId,
        tree: DecalsTree
      },
```

![screen shot 2016-04-07 at 5 31 32 pm](https://cloud.githubusercontent.com/assets/2285931/14367546/99cc67ee-fce6-11e5-987c-386f37d51502.png)

I'm not entirely sure what the `undefined property`` means here, so I'm stuck. I'd really like to get a local version so I can try to help getting the Examine page running again, since a number of users have already commented on not being able to see it.

**Comment by willettk** [comment]
@CKrawczyk - did you have this problem testing out the local version of GZ?

**Comment by CKrawczyk** [comment]
It loads fine locally on my computer.

**Comment by murraycu** [comment]
surveys.decals_dr2 seem to be the undefined thing. Maybe something about your configuration (production/test/whatever) means you are using a version of Config.surveys without that.

**Comment by willettk** [comment]
I've never set the configuration explicitly (and am not sure how to, although details would be helpful).

**Comment by bamford** [comment]
Probably fixed in e32d7dad.

**Comment by CKrawczyk** [comment]
This is fixed now.
